app/api/V1/views/login.py

```diff
@@ -37,7 +37,7 @@ def post(self):
 		
 		user = User.get_one_user(self, email)
 
-		if email and user == "User not found":
+		if user == "User not found":
```

Comment:
what does this line of code do?

Reply:
It checks if there is a user already that has already been registered